### ISSUE TITLE
当扩展Block定义了"extensions": string[]对象后无法加载

### ISSUE THREAD

**Issue by java30433** [EurekaScratch/eureka#27]
问题描述：定义一个Block，含"extensions": string[]对象，加载时发生错误
Cannot read properties of undefined (reading 'includes')
问题定位：src/loader/loader.ts line:611
```
if (!('extensions' in converted.json)) converted.json = [/*'scratch_extension'*/];
if (!converted.json.extensions.includes(extension)) { //json.extensions is undefined
```
解决办法：修改为
`if (!('extensions' in converted.json)) converted.json.extensions = [/*'scratch_extension'*/];`

**Comment by java30433** [comment]
另：考虑添加 turbowarp.cn ?

**Comment by SimonShiki** [comment]
fixed in aba036a
